### PR TITLE
Updated Logos

### PR DESCRIPTION
**Pull request by yatil**
I hope this updates the new logo, I wasn’t able to compile/test it locally. I only got a white page using `grunt serve`. (I hope this is the right branch to merge into, it was very hard to find where the development was going on.)

Rel. #305

### CODE REVIEW

**Comment by rvantonisse** [comment]
Thanks yatil, yes correct branch, but might even merge as hotfix into master since this is a brand update. Checking this out.

**Comment by rvantonisse** [comment]
@yatil you might need to double install bower components. simply run `npm install` twice for ease.

You can then serve with `npm start` or build with `npm run build`

File: app/views/navigation.html [hunk 1]
```diff
@@ -3,16 +3,15 @@
     <div class="branding">
       <div class="nav navbar-nav navbar-right">
         <span class="w3c"><a href="http://w3.org/">
-          <img alt="W3C" width="85" height="50" src="images/w3c.png">
+          <img alt="W3C" width="110" height="75" src="images/w3c.png">
```
Comment (rvantonisse) [inline app/views/navigation.html]: Undo height changes please, this breaks current design, or is there a reason it needs to be this size?
(And by all means keep the right aspect ratio for the logo)

File: app/views/navigation.html [hunk 2]
```diff
@@ -3,16 +3,15 @@
...
         </a></span>
         <span class="wai"><a href="http://w3.org/WAI/">
-          <img alt="Web Accessibility Initiative" width="194" height="48"
+          <img alt="Web Accessibility Initiative" width="180" height="75"
```
Comment (rvantonisse) [inline app/views/navigation.html]: Same here please undo height changes.

**Comment by rvantonisse** [comment]
Possibly totally redundant but these are the logos changed:
W3C

WAI

**Comment by yatil** [comment]
@slhenry I updated the logo and changed some of the colors to better work with the new design. No time  for a comprehensive update, but I think it’s also not high priority at this point.

Screen shot that shows the changes:

![Screen Shot(https://user-images.githubusercontent.com/58862/70515566-d80a6d00-1b35-11ea-950a-7b4e35f6ae73.png)

@rvantonisse Feel free to merge however you feel it should be merged once @slhenry has approved it. I don’t feel strongly. (Feel free to squash the merge, no need to keep the pngs in history :-D)

**Comment by slhenry** [comment]
+1

**Pull request merged**